### PR TITLE
Move IsThemeBackgroundPartiallyTransparent to Interop UxTheme

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Move IsThemeBackgroundPartiallyTransparent to Interop UxTheme

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2456)

### CODE REVIEW

File: src/Common/src/Interop/UxTheme/Interop.IsThemeBackgroundPartiallyTransparent.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    public static partial class UxTheme
+    {
+        [DllImport(Libraries.UxTheme)]
```
Comment (hughbe) [inline src/Common/src/Interop/UxTheme/Interop.IsThemeBackgroundPartiallyTransparent.cs]: nit: add `ExactSpelling = true`

File: src/System.Windows.Forms/src/System/Windows/Forms/VisualStyles/VisualStyleRenderer.cs [hunk 2]
```diff
@@ -987,7 +987,7 @@ public HitTestCode HitTestBackground(IDeviceContext dc, Rectangle backgroundRect
         /// </summary>
         public bool IsBackgroundPartiallyTransparent()
         {
-            return (SafeNativeMethods.IsThemeBackgroundPartiallyTransparent(new HandleRef(this, Handle), part, state));
+            return UxTheme.IsThemeBackgroundPartiallyTransparent(this, part, state).IsTrue();
```
Comment (hughbe) [inline src/System.Windows.Forms/src/System/Windows/Forms/VisualStyles/VisualStyleRenderer.cs]: could add a test e.g.

````cs
public void VisualStyleRenderer_IsBackgroundPartiallyTransparent_Invoke_ReturnsExpected()
{
    var renderer = new VisualStyleRenderer("BUTTON", 0, 0);
    Assert.False(renderer. IsBackgroundPartiallyTransparent);
}
```

File: src/Common/src/Interop/UxTheme/Interop.IsThemeBackgroundPartiallyTransparent.cs (hunk 1 above)
Comment (RussKie) [reply src/Common/src/Interop/UxTheme/Interop.IsThemeBackgroundPartiallyTransparent.cs]: I've been documenting various nuances about p/invokes here - https://github.com/dotnet/winforms/wiki/Useful-guidelines#q-when-to-use-the-exactspelling-field-and-an-explicit-w-in-dllimportattributes
You may find them useful.

Feel free to tell me to add more things that you think may be useful.

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/VisualStyles/VisualStyleRendererTests.cs [hunk 3]
```diff
@@ -569,5 +569,12 @@ public void VisualStyleRenderer_SetParameters_InvalidElement_ThrowsArgumentExcep
             var renderer = new VisualStyleRenderer(VisualStyleElement.Button.PushButton.Hot);
             Assert.Throws<ArgumentException>(null, () => renderer.SetParameters(element));
         }
+
+        [Fact]
+        public void VisualStyleRenderer_IsBackgroundPartiallyTransparent_Invoke_ReturnsExpected()
+        {
+            var renderer = new VisualStyleRenderer("BUTTON", 0, 0);
+            Assert.False(renderer.IsBackgroundPartiallyTransparent);
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/VisualStyles/VisualStyleRendererTests.cs]: This is a method call, not a property

**Pull request merged**